app/test_main.py

```diff
@@ -1,3 +1,13 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_different_between_the_same_animals_age() -> None:
+    assert get_human_age(28, 28) == [3, 2]
+
+
+def test_negative_values() -> None:
+    assert get_human_age(-1, -1) == [0, 0]
+
+
+def test_large_values() -> None:
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
did u read checklist ? write more tests
use mark.parametrize for this
test cases for example
![image](https://user-images.githubusercontent.com/103201010/220427576-5f1718b6-331d-4188-a05e-1ae3c828da1b.png)